app/views.py

```diff
@@ -0,0 +1,80 @@
+'''Bucket list application'''
+
+from flask import Flask, flash, session, abort, render_template, url_for, redirect, request
+from app_functionality import User, BucketList
+
+app = Flask(__name__)
+
+
+@app.route('/')
+def index():
+    '''Returns rendered Homepage(Index page) of the app'''
+    return render_template('index.html')
+
+
+@app.route('/register', methods=['GET', 'POST'])
+def register():
+    '''Registers a new user, allowing them to use app'''
+
+    if request.method == 'POST':
+        email = request.form['email']
+        user_name = request.form['user_name']
+        password = request.form['password']
+        if user_name and password and email:
+            user = User(email, user_name, password)
+            user.register_user()
+            return redirect(url_for('create'))
```

Comment:
when you have the return you do not need the `else:` the code would have stopped executing